### PR TITLE
Revert "Update sewer-cli.md"

### PR DESCRIPTION
**Pull request by mmaney**
Reverts komuw/sewer#217

I had to scratch my head to figure this one out.  It's rooted in the half-assed "pages" feature github has, and ought to have been `[Aliasing](docs/Aliasing)`, as most of the core docs have been migrated to the web pages view.   Which I still think is better than having wiki pages that aren't versioned in the repo, but it still leaves a lot to be desired.

### CODE REVIEW

**Comment by komuw** [comment]
@mmaney If I go to https://github.com/komuw/sewer/blob/master/docs/sewer-cli.md and then I click on the aliasing link, it takes me to https://github.com/komuw/sewer/blob/master/docs/Aliasing which returns a 404.  


The way it was implemented in https://github.com/komuw/sewer/pull/217 was working okay.

**Comment by mmaney** [comment]
On Sat, Jan 23, 2021 at 09:12:00AM -0800, Komu Wairagu wrote:

> @mmaney If I go to https://github.com/komuw/sewer/blob/master/docs/sewer-cli.md and then
> I click on the aliasing link, it takes me to
> https://github.com/komuw/sewer/blob/master/docs/Aliasing which returns a 404.

Yes, IF you're viewing it as an md file in the repo.  But that link is
broken if you're viewing it through github pages.

https://komuw.github.io/sewer/sewer-cli  (as linked from the README, eg. 
Also arrrrrgh - what happened to the table rendering there?  github
rendering fail?)

For better or worse most [newer] things are using pages links.  There
is, so far as I've found, no good solution to this dichtomy.  It's
actually making me reconsider using the wiki for documentation...  or
getting off github's half-assed solutions for docs, as so many other
projects do, but I don't really like ReadTheDocs a whole lot, either.

...and although it feels like I've done boatloads of work on these in
the last year, I just found another boatload of stuff that's still in
the old style.  :-(

Sometimes I just hate github.  Have I ever mentioned that getting
involved with sewer was what made me re-join for the third (fourth?)
time?  It always decays this way, from "oh, this is better than I
remember", to a peak "nice new feature", through disillusionment, to
deleting all my repos and closing the account again.  Because github is
basically an impediment to the git repository itself, however much it
enhances the social media feels.

Just ventin' ... and sewer is rather more of an incentive to stick it
out, in terms of having visible users.  <wink>

I wonder if the custom 404 support is sufficient to do more than
explain this problem?  Not that this would be more than a bandaid, of
course...

-- 
No individual firm, no individual capitalist is now thought to have power;
that the market is subject to skilled and comprehensive management
is unmentioned even in most economic teaching.  Here the fraud.  -- JKG
